docs/alphafold_transcript_coverage.md

```diff
@@ -0,0 +1,620 @@
+# Abstract
+
+The AlphaFold/AlphaFold2 database from EMBL-EBI is comprised of a single protein structure per gene symbol. These are named based on UniProt accessions with identical sequence to each AlphaFold structure, allowing precise mapping of sequence-based information to the protein tertiary structures. While useful, there is enormous clinical value in building structures from alternative transcripts for each gene symbol, where we have knowledge that these are protein-coding and expressed. Often, alternate transcripts are of significant relevance in specific tissues, such as the brain or immune cells and of specific interest to scRNA-Seq experiments. We explore the coverage of the protein-coding space in the human genome compared to the AlphaFold structures and show that 11.8% of actionable ClinVar variants are currently not represented and contained within alternate transcripts.  
+
+
+# Steps
+
+
+## Files downloaded
+
+MANE Select transcripts  
+
+-   MANE.GRCh38.v0.95.summary.txt.gz
+-   <https://ftp.ncbi.nlm.nih.gov/refseq/MANE/MANE_human/current/MANE.GRCh38.v0.95.summary.txt.gz>
+-   Accessed 10/11/2021
+
+AlphaFold2 Homo sapiens Structures  
+
+-   UP000005640\_9606\_HUMAN.tar
+-   <https://ftp.ebi.ac.uk/pub/databases/alphafold/UP000005640_9606_HUMAN.tar>
+-   Accessed 9/11/2021
+
+GENCODE protein-coding transcripts  
+
+-   gencode.v39.pc\_transcripts.fa
+-   <http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/latest_release/gencode.v39.pc_transcripts.fa.gz>
+-   Accessed 20/1/22
+
+GENCODE protein-coding translations  
+
+-   gencode.v39.pc\_translations.fa
+-   <http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/latest_release/gencode.v39.pc_translations.fa.gz>
+-   Accessed 20/1/22
+
+GENCODE comprehensive gene annotation (gtf format)  
+
+-   gencode.v39.annotation.gtf
+-   <https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_39/gencode.v39.annotation.gtf.gz>
+-   Accessed 20/1/22
+
+ClinVar variant summary table  
+
+-   variant\_summary.txt.gz
+-   <https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz>
+-   Accessed 25/1/22
+
+
+## Unpack the AlphaFold2 transcripts
+
+Unzip the AlphaFold2 structures into a folder. In my case, `~/data/af2/`  
+
+    tar -xvf UP000005640_9606_HUMAN.tar
+
+
+## Get the UniProt ID&rsquo;s corresponding to AF2 structures
+
+Quick shell command to parse the names from the AF2 structure filenames.  
+
+    ls | awk -F"-" '{print $2}' | sort | awk '!a[$0]++' > uniprots.txt
+
+20,504 unique UniProt IDs to use.  
```

Comment:
Is this just a general comment of how many you found in this example, or is will this stay permanent? If the latter, I'd expand and explain that a bit more formally.

Reply:
Thanks! I'll expand on this in a separate PR after moving pages deployment to main branch